tests/test_consumer.py

```diff
@@ -219,17 +219,8 @@ def test_multistorage_strategy(
         for message in messages:
             strategy.submit(message)
 
-        with assert_changes(
```

Comment:
Remove this assertion since it was testing internal arroyo functionality, which was changed in https://github.com/getsentry/arroyo/commit/e6c6ec1a6d999ed8652e240e714605abc37a91f7